content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,65 @@
+Title: A tour of modern frontend with React and Flux
+Date: 2015-08-18
+Short_summary: React and Flux have swept the frontend in the last year or so. Let's look at it.
```

Comment:
I guess either "Let's look at them" or "have swept the front-end development world" would work better